server.js

```diff
@@ -46,6 +46,9 @@ http.createServer( function( request, response ) {
 	const args = params.query || {};
 	if ( typeof args.webp === 'undefined' ) {
 		args.webp = !!( request.headers && request.headers['accept'] && request.headers['accept'].match( 'image/webp' ) );
+	} else {
+		// args.webp will always be a string at this point, so lets convert it to a boolean to not break future conditionals.
+		args.webp = (args.webp == true);
```

Comment:
Sorry folks I missed this.

So, if we expect `args.webp == true` to return `false` in the case of `0` being passed, then I don't see why the later checks would also be failing. Should this not be something more like `args.webp == "true" || args.webp == "0"`.

Reply:
The idea was to not break the existing workaround but also support using a real truthy value.

Reply:
<img width="113" alt="image" src="https://user-images.githubusercontent.com/442115/198306815-1474959a-b699-48e9-b6b9-c70b08dc073e.png">
just a note that the string `true` does not loosely-equal the boolean value `true`

Reply:
Ok, my bad, this I did not expect! I assumed that `if ( "true" == true )` is identical to `if ( "true" )` but, apparently that isn't the case.

So, just to confirm: we're expecting `args.webp === "0"`. Currently `if ( args.webp ) // when args.webp = "0"` evaluates to `true`, so there's effectively no way to turn off webp. `if (  args.webp == true )  // when args.webp = "0"` evaluates to `false`. I don't know if it's because it's a Friday but this is totally breaking my brain :P